.github/workflows/pr-lint.yml

```diff
@@ -0,0 +1,21 @@
+name: PR Lint
+on: pull_request
+
+jobs:
+  markdownlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: install dependencies
+        run: |
+          mkdir markdownlint-cli && cd markdownlint-cli
```

Comment:
instead of putting all the logic here, I rather we created another tool under tools with a package.json and just “npm install” it.

Reply:
Nice! I like this pattern. It's slightly better than @MrArnoldPalmer's suggestion of keeping a package.json at the package root, and yet satisfies his comment. I'll change this up.